mock_model.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional
+import random
+
+
+class MockModel:
```

Comment:
I support this mocking framework, but eventually you might want an external mock application that feeds the backend data to process